test/boost/cql_query_test.cc

```diff
@@ -357,7 +357,7 @@ SEASTAR_TEST_CASE(test_tuple_elements_validation) {
 SEASTAR_TEST_CASE(test_list_of_tuples_with_bound_var) {
     return do_with_cql_env_thread([](cql_test_env& e) {
         e.execute_cql("create table cf (pk int PRIMARY KEY, c1 list<frozen<tuple<int,int>>>);").get();
-        BOOST_REQUIRE_THROW(e.prepare("update cf SET c1 = c1 + [(?,9999)] where pk = 999;").get0(), exceptions::invalid_request_exception);
+        e.prepare("update cf SET c1 = c1 + [(?,9999)] where pk = 999;").get0();
```

Comment:
Again, it's weird to see a test that does an update without reading back and checking that the update actually worked.